packages/smooth_app/ios/Runner/AppDelegate.swift

```diff
@@ -9,6 +10,18 @@ import Flutter
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
     GeneratedPluginRegistrant.register(with: self)
+      
+      WorkmanagerPlugin.setPluginRegistrantCallback { registry in
```

Comment:
You can remove this part, since we won't use it

Reply:
Not having a mac, I avoided editing the ios specific files,

Reply:
No problem for that, while i made some changes pushing them before you do more changes in it 
Thanks for the rescue here :)

Reply:
Now you can do the commits @g123k , I am working on making the code more modular